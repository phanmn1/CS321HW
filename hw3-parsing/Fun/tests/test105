// numbers aren't allowed as the first character
// of a variable name
int 2count = 0;

int fact(int n) {
  count = count + 1;
  if (n<1)
    return 1;
  else
    return n * fact(n-1);
}

int x = y;
int y = 1;

void main() {
  print fact(7);
  print fact(3);
  print fact(count);
}

