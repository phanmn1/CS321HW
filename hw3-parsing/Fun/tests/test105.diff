1c1
< Invalid syntax at (1,3), int
---
> Invalid syntax at (5,3), 2
