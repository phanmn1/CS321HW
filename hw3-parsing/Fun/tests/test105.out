Invalid syntax at (5,3), 2
